samwise.js

```diff
@@ -2,7 +2,7 @@
  * Autoplay plugin - Automatically advance slideshow after N seconds
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!